lesson-5/Chen_Homework Source Files/style.css

```diff
@@ -0,0 +1,38 @@
+body {
+  font-family: "Work Sans", sans-serif;
+}
+.container {
+	position:fixed;
+	top:50%;
+	left:50%;
+	transform: translate(-50%,-50%);
+}
+.colorBox{
+	background-color: #f7f2f2;
+	
+	padding:20px;
+	width:235px;
+
+}
+
+a{background-color:white;
+	display:block;
+	text-align:center;
+	border-radius:5px;
+	width:200px;
+	height:30px;
```

Comment:
Instead of using `height` here, we can just give the `a` some padding. If we give it the same padding the whole way around, the text will always appear directly in the middle.